src/main/java/com/exasol/adapter/adapternotes/SchemaAdapterNotesJsonConverter.java

```diff
@@ -67,6 +67,16 @@ public String convertToJson(final SchemaAdapterNotes schemaAdapterNotes) {
         return builder.build().toString();
     }
 
+    private static void checkKey(final JsonObject root, final String key, final String schemaName)
```

Comment:
You introduced a new private method that I don't see any usage of in the changeset. Is this unused?

Reply:
It's not new, It was just reorganized by the formatter.